src/DependencyInjection/ProophEventStoreExtension.php

```diff
@@ -176,7 +176,7 @@ private function loadEventStore(string $name, array $options, ContainerBuilder $
                 $repositoryClass = $repositoryConfig['repository_class'] ?? $repositoryName;
 
                 if (! class_exists($repositoryClass)) {
-                    throw new RuntimeException(\sprintf(
+                    throw new RuntimeException(sprintf(
```

Comment:
Why was the `\` backslash removed from `sprintf`? This gives a minimal [performance gain](https://github.com/Roave/FunctionFQNReplacer#benchmark).

Reply:
I've read somewhere, that the performance difference will be gone in future php version and you shouldn't use this except when needed (but I can't find the sources for this again at the moment). Currently we don't do this within all the prooph components, so even if my statement above would be invalid, let's keep it consistent.

Reply:
and tools like cs-fixer can always change it in entire project